ansible/roles/concord-demo/files/concord.conf

```diff
@@ -6,5 +6,5 @@ respawn
 
 env LD_LIBRARY_PATH=/usr/local/lib
 env MESOS_NATIVE_JAVA_LIBRARY=/usr/local/lib/libmesos.so
-exec java -DMESOS_NATIVE_JAVA_LIBRARY=/usr/local/lib/libmesos.so -cp /usr/local/share/concord/concord-scheduler-assembly-1.0.jar com.concord.scheduler.Scheduler -l 127.0.0.1:11211 -t 127.0.0.1:11219 -m zk://127.0.0.1:2181/mesos -c zk://127.0.0.1:2181/concord -n concord
+exec java -DMESOS_NATIVE_JAVA_LIBRARY=/usr/local/lib/libmesos.so -cp /usr/local/share/concord/concord-scheduler-assembly-1.0.jar com.concord.scheduler.Scheduler -l 127.0.0.1:11211 -m zk://127.0.0.1:2181/mesos -k zk://127.0.0.1:2181/kafka -c zk://127.0.0.1:2181/concord -n concord
```

Comment:
Agreed, eventually I want to make wider changes, will scope this feature when we get there.